packages/signals_flutter/lib/src/signals/core/computed.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+
+import '../../../signals_flutter.dart';
+
+final _signals = <WeakReference<Element>, Map<int, Computed>>{};
+
+/// Create and watch a computed signal in a build method
+///
+/// ```dart
+/// class MyWidget extends ... {
+///  @override
+///  Widget build(BuildContext context) {
+///    final count = useSignal(context, 0);
+///    final isEven = useComputed(context, () => count.value % 2 == 0);
+///    return Row(
+///     children: [
+///       IconButton(icon: Icon(Icons.remove), onPressed: () => count.value--),
+///       Text('$count event=$isEven),
+///       IconButton(icon: Icon(Icons.add), onPressed: () => count.value++),
+///    ],
+///   );
+///  }
+/// }
+/// ```
+Computed<T> useComputed<T>(
+  BuildContext context,
+  T Function() value, {
+  String? debugLabel,
+  SignalEquality<T>? equality,
+}) {
+  assert(context is Element);
+  final key = (value, debugLabel).hashCode;
```

Comment:
Wouldn't it be safer to include `T` in the key? This way, `useSignal<String?>(null)` and `useSignal<int?>(null)` won't collide, for example.